### PR TITLE
Fix/Align initial screen style with the rest of the client

### PR DESCRIPTION
**Pull request by mhrvth**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

Initial screens apply the selected color configuration.

**Tests performed by the developer**:

Ran the client.

**Tips for QA testing**:

Please test on Windows specifically because I couldn't.

### CODE REVIEW

**Comment by mhrvth** [comment]
[sc-22050]

**Comment by gerald-hb** [comment]
Interface on Initial setup on Windows looks fine but still got a line on the hummingbot image above the output pane.

image

File: hummingbot/client/ui/__init__.py [hunk 1]
```diff
@@ -77,30 +68,30 @@ def show_welcome():
     data. Please store this password safely since there is no way to reset it.
 
         """,
-        style=dialog_style).run()
+        style=style).run()
 
 
-def login_prompt():
+def login_prompt(style):
```
Comment (aarmoa) [inline hummingbot/client/ui/__init__.py:74]: We should add type hinting to the new parameter

**Review by aarmoa — CHANGES_REQUESTED** [review]

**Comment by mhrvth** [comment]
> Interface on Initial setup on Windows looks fine but still got a line on the hummingbot image above the output pane.
> 
> image

Hi, how does it look like in the development branch?

**Comment by gerald-hb** [comment]
Hi @mhrvth Apologies for the confusion. The test earlier was on a Linux terminal. I can confirm that it is fixed now
image

I am unable to run a test on Windows (gitbash) due to another issue.
image

**Comment by gerald-hb** [comment]
Since there is an outstanding with Git bash (Windows) I have run another test using WSL (Ubuntu 18.04). No issue on UI
image

Test Summary
-Install the feature branch and start the client: :white_check_mark: 
-Check the Hummingbot style: :white_check_mark: 		
-Check the text style in the terms of use: :white_check_mark: 			
-Check the style on the password setup: :white_check_mark:  			
-Verify Hummingbot in the client: :white_check_mark: 
-Restart the client and verify the style in the password section: :white_check_mark:

**Comment by mhrvth** [comment]
> Hi @mhrvth Apologies for the confusion. The test earlier was on a Linux terminal. I can confirm that it is fixed now image
> 
> I am unable to run a test on Windows (gitbash) due to another issue. image

This looks like an unrelated error coming from the gateway / docker. Does it also happen on the development branch or just here?

**Review by RC-13 — APPROVED** [review]
LGTM!

Test summary:
- Check the styling for the Hummingbot startup logo for both Mac and Linux
- Check the styling of the dialog box for the terms of use for both Mac and Linux
- Check the styling of the password dialog box for both Mac and Linux

**Review by aarmoa — APPROVED** [review]
Changes look good to me

**Comment by mhrvth** [comment]
Created a PR in the Foundation repo: https://github.com/hummingbot/hummingbot/pull/5303

**Pull request closed without merging**